package.json

```diff
@@ -45,46 +46,56 @@
     ]
   },
   "dependencies": {
-    "@jupyterlab/application": "^2.0.0",
-    "@jupyterlab/apputils": "^2.0.0",
-    "@jupyterlab/codeeditor": "^2.0.0",
-    "@jupyterlab/coreutils": "^4.0.0",
-    "@jupyterlab/docmanager": "^2.0.0",
-    "@jupyterlab/docregistry": "^2.0.0",
-    "@jupyterlab/filebrowser": "^2.2.4",
-    "@jupyterlab/fileeditor": "^2.0.0",
-    "@jupyterlab/launcher": "^2.2.5",
-    "@jupyterlab/mainmenu": "^2.2.3",
-    "@jupyterlab/services": "^5.0.0",
-    "@jupyterlab/settingregistry": "^2.0.0",
-    "@jupyterlab/statedb": "^2.0.0",
-    "@jupyterlab/ui-components": "^2.2.3",
-    "@lumino/coreutils": "^1.3.1",
-    "@lumino/disposable": "^1.2.0",
-    "@lumino/domutils": "^1.1.3",
-    "@lumino/messaging": "^1.2.3",
-    "@lumino/widgets": "^1.8.0",
+    "@jupyterlab/application": "^3.0.0",
```

Comment:
if there were really no code changes (which is good), could these all be `^2.0.0 || ^3.0.0`?